.expeditor/config.yml

```diff
@@ -110,19 +110,19 @@ subscriptions:
   # the omnibus/docker/gem chain
   - workload: artifact_published:unstable:chef:{{version_constraint}}
     actions:
-      # - trigger_pipeline:docker/build
+      - trigger_pipeline:docker/build
       - trigger_pipeline:macos_universal_package
-  # - workload: artifact_published:current:chef:{{version_constraint}}
-  #   actions:
-  #     - bash:.expeditor/promote-docker-images.sh
+  - workload: artifact_published:current:chef:{{version_constraint}}
```

Comment:
With not doing omnibus builds this workflow will not work. We will ahve to likely remove this value here and add it here:

  - workload: project_promoted:{{agent_id}}:*
    actions:
      - built_in:promote_artifactory_artifact
      - bash:.expeditor/promote-docker-images.sh